### DISCUSSION TITLE
Whats the difference between `react-instantsearch-hooks` and `react-instantsearch-hooks-web`?

### DISCUSSION THREAD

**Discussion by christiankaindl** [algolia/instantsearch#5331] [Q&A] [answered]
**Why are there 2 packages with almost the same name? What's the difference between them?**

Both state that they are an open-source UI library for React, with only minimal differences in wording. I find this really confusing.

`react-instantsearch-hooks-web` -> "React InstantSearch Hooks Web is an open-source React library that lets you create an instant search result experience using Algolia’s search API. It is part of the InstantSearch family"

`react-instantsearch-hooks` -> "React InstantSearch Hooks is an open-source UI library for React that lets you quickly build a search interface in your front-end application"

[Docs say](https://www.algolia.com/doc/api-reference/widgets/react-hooks/) use `react-instantsearch-hooks-web` in their examples, but why?

**Comment by Haroenv** [comment, marked as answer]
React InstantSearch Hooks is the underlying library that exports renderless components and hooks, this is useful for web applications, but also for react native for example. React InstantSearch Hooks Web is the implementation for the web, so it includes the components and rendering that works with DOM.

I see how this could be confusing. Was there a specific spot you expected the difference between the packages to be explained where we can add it?

**Reply by christiankaindl** [reply]
Thanks for the explanation :)

I'd expect the readme of `react-instantsearch-hooks` to include some sort of description of what makes it different from the `*-web` package, with clear usage guidance. Something like this:
> Note: `react-instantsearch-hooks` exports renderless components and hooks which can be used for both web and React Native. If you are using React in a web project, we recommend using the package [`react-instantsearch-hooks-web`](https://www.npmjs.com/package/react-instantsearch-hooks-web) instead, as it includes complete components that render to the DOM.

and then in the `*-web` package:
> Note: If you are working with React Native, check out `react-instantsearch-hooks` instead.

Feel free to take these or write them differently, but _something_ should be in the readme imo :)
